BUILD_macos.md

```diff
@@ -70,6 +70,11 @@ PRs with corrections and updates are welcome!
   brew install libfido2
   ```
 
+* To install `yarn` for building the UI
+  * Install Node.js from the [Node.js Releases/Download Page](https://nodejs.org/en/download/)
+  * Run `sudo corepack enable`
+  * Run `corepack prepare yarn@stable --activate`
```

Comment:
I believe that `corepack enable yarn` should be sufficient.

Reply:
Much simpler. Updated.